app/components/procedure/invitation_with_typo_component.rb

```diff
@@ -0,0 +1,21 @@
+class Procedure::InvitationWithTypoComponent < ApplicationComponent
+  def initialize(maybe_typo:, url:, params:, title:)
```

Comment:
```suggestion
  def initialize(maybe_typo:, url:, params:, title:)
```

passer une proc en params me parait très compliqué pour ce qu'on veut faire.
Y'a pas moyen de simplifier ?

Reply:
https://github.com/demarches-simplifiees/demarches-simplifiees.fr/pull/10524/commits/4771c45bce3de89e08dba42f1786e931c8922377